lib/hammer_cli_foreman_docker/docker_container.rb

```diff
@@ -74,7 +74,7 @@ class StatusCommand < HammerCLIForeman::InfoCommand
       action :power
 
       output do
-        field :running, _("Running"), Fields::Boolean
```

Comment:
I'd prefer to to keep the Boolean type and transform data at the input to the format that is expected. 
If you are interpreting the data on your own there is a risk the output will be inconsistent and export to the other formats (YAML, JSON) won't work as expected.